modules/plugin/mongodb/src/test/java/org/geotools/data/mongodb/MongoDataStoreTest.java

```diff
@@ -198,4 +206,27 @@ private void clearSchemaStore(MongoDataStore mongoStore) {
         }
         mongoStore.cleanEntries();
     }
+
+    public void testSortBy() throws Exception {
```

Comment:
This is an online test right? I would also add a test for DESCENDING.

Reply:
Forget about this just notice that you have that test on ``MongoFeatureSourceTest``.